ci-operator/step-registry/openstack/deprovision/dnsrecords/openstack-deprovision-dnsrecords-commands.sh

```diff
@@ -7,10 +7,13 @@ set -o pipefail
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 export AWS_DEFAULT_REGION=us-east-1
 export AWS_DEFAULT_OUTPUT=json
-export AWS_PROFILE=profile
```

Comment:
@pierreprinetti why do we need this? CI fails if we drop that.

Reply:
Sorry for the ping, I thought you added it but by looking further in Git you're not the author of that.